File: 3sem/poo/Aula01-Codigo/Java/Exemplo04-Heranca/src/exemplo4heranca/Animal.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package exemplo4heranca;

/**
 *
 * @author Junio
 */
public class Animal extends Object{
    String sNome;
    Animal(String umNome){
        sNome = umNome + "!";
    }

    public String getsNome() {
        return sNome;
    }
}

File: 3sem/poo/Aula01-Codigo/C++/Exemplo04-Heranca/Animal.cpp
#ifndef animal
#define animal
#include <string>

using namespace std;
class Animal {
   private:
    string sNome;

   public:
    Animal(string umNome){
        sNome = umNome;
    };          
    string getNome() {
        return sNome;
    };
};
#endif
